### PR TITLE
Fix for issue 6639

### LINKED ISSUE #6639: Enable one column libreoffice panels
**Is your suggestion for improvement related to a problem? Please describe.**
I am using jabref with LibreOffice, the connect panel is 3x3 grid with small icons on top. It is very inconvenient when citing references because the 3x3 grid makes the main panel of jabref narrower.

**Describe the solution you'd like**
would it be possible that this LibreOffice plugin is 1 column, i.e., a side panel.

**Additional context**
jabref

### PR DESCRIPTION
**Pull request by ddunig2**
-Changed the layout of the side panel to flow layout to keep some of its items from readable.
-fixes #6639 

image

- [x] Change in CHANGELOG.md described (if applicable)
- [ ] Tests created for changes (if applicable)
- [x] Manually tested changed features in running JabRef (always required)
- [x] Screenshots added in PR description (for UI changes)
- [ ] [Checked documentation](https://docs.jabref.org/): Is the information available and up to date? If not created an issue at <https://github.com/JabRef/user-documentation/issues> or, even better, submitted a pull request to the documentation repository.

### CODE REVIEW

**Comment by ddunig2** [comment]
I can add the changes to the changlog, I just wanted to know if it goes at the top in the change section or at the bottom, or does it even matter?

**Comment by koppor** [comment]
In case you find a similar CHANGELOG entry, just add it next to it. Otherwise at the bottom.

File: src/main/java/org/jabref/gui/openoffice/OpenOfficePanel.java [hunk 1]
```diff
@@ -264,19 +266,19 @@ private void initPanel() {
...
-        VBox row1 = new VBox();
-        VBox row2 = new VBox();
-        VBox row3 = new VBox();
-        row1.getChildren().addAll(setStyleFile, pushEntries, pushEntriesInt);
-        row2.getChildren().addAll(pushEntriesAdvanced, pushEntriesEmpty, merge);
-        row3.getChildren().addAll(manageCitations, exportCitations, settingsB);
-        HBox hbox1 = new HBox();
-        hbox1.getChildren().addAll(row1, row2, row3);
-        hbox1.getChildren().forEach(btn -> HBox.setHgrow(btn, Priority.ALWAYS));
+
```
Comment (koppor) [inline src/main/java/org/jabref/gui/openoffice/OpenOfficePanel.java:269]: Please no extra whitespace 😇

Comment (ddunig2) [reply src/main/java/org/jabref/gui/openoffice/OpenOfficePanel.java:269]: so lines 269, 275 and 279 can all go?

**Comment by tobiasdiez** [comment]
Thanks a lot for your contribution! I'll merge now.

**Pull request merged**